.prettierrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     "tabWidth": 2,
-    "useTabs": false
+    "useTabs": false,
+    "endOfLine": "auto"
```

Comment:
Same here. Please change these back.

Reply:
Ahh, this was partially my fault - I had default line endings as `CRLF` instead, so a bunch of errors were showing up.